docs/arch/adr-29.md

```diff
@@ -0,0 +1,31 @@
+# ADR 29: Video Player
+
+Created: Dec 8, 2020
+
+## Context
+
+Future projects using the `front-end-monorepo` will require a video player. Some projects may simply need a video to play. Some projects will require users to interact with a video file; such as make annotations on the video to mark locations or item sizes. The video player must be highly customizable, able to be used in a React code base project and support all major browsers.
+
+## Decision
+
+We will implement `react-player` [Github](https://github.com/CookPete/react-player).
+Demo Page: [Demo](https://cookpete.com/react-player/)
+
+### Media
+
+`react-player` is a React component for playing a variety of URLs, including file paths, YouTube, Facebook, Twitch, SoundCloud, Streamable, Vimeo, Wistia, Mixcloud, and DailyMotion. [Supported Media](https://github.com/CookPete/react-player#supported-media)
+
```

Comment:
Even though the player supports externally hosted videos at these sites, I think we should have a note here about whether or not we'll support that. Even though we do allow subject locations to be external (hosting on the researcher's servers), I'm not sure we should allow a youtube (or other) hosted location. We do validations for particular mime types, like mp4, as well